Dockerfile

```diff
@@ -4,13 +4,14 @@ RUN apk --no-cache add ca-certificates make
 
 WORKDIR /app
 
-COPY package.json package-lock.json ./
+COPY package.json ./
```

Comment:
Shall we copy `yarn.lock` too here? 
```suggestion
COPY package.json yarn.lock ./
```

Reply:
✅  added in [dc9448c](https://github.com/carryall/go-google-scraper-challenge/pull/53/commits/dc9448c8e3dd3889c01321cead090377cc56fcc1)